Backend/src/main/java/com/SYSC4806_Group13/SYSC4806_Project/Controllers/CartItemRestController.java

```diff
@@ -0,0 +1,109 @@
+package com.SYSC4806_Group13.SYSC4806_Project.Controllers;
+import com.SYSC4806_Group13.SYSC4806_Project.Model.CartItem;
+import com.SYSC4806_Group13.SYSC4806_Project.Model.CartItemRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@RestController
+public class CartItemRestController {
+    @Autowired
```

Comment:
Probably want to do constructer dependency injection instead of this.
Something like:
```
    private final CartItemRepository repo;

    public CartItemRestController(CartItemRepository repo) {
        this.repo = repo;
    }
```